crates/nix-ninja-task/src/main.rs

```diff
@@ -40,19 +41,16 @@ fn main() -> Result<()> {
 
     println!("NIX_BUILD_TOP {}", env::var("NIX_BUILD_TOP")?);
```

Comment:
These are diagnostic messages that should probably go into stderr rather than stdout.